libs/api-catalogue/elastic/src/lib/types/types.model.ts

```diff
@@ -0,0 +1,41 @@
+interface ShardsResponse {
```

Comment:
Is this file automatically generated?

Reply:
No as of now the definitions are not available from elasticsearch, but this based on TypeScript support description here: https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/typescript.html

Reply:
I see, then it should be similar to this one used on the api/elastic-search https://github.com/island-is/island.is/blob/master/libs/api/content-search/src/types/index.ts I wonder if we could share some part of this file somewhere instead of typing it locally on each project

Reply:
Yes, I agree, would be nice to have a lib or something of that sorts for the shared types :)

Reply:
So should we maybe create a island-ui/elastic library that other libraries and projects can reuse these basics types?

Reply:
Since it's only types, it might be confusing to create a whole lib for it. I wonder if it could be instead inside `libs/island-shared/types/elastic.d.ts` for example

Reply:
Sounds good, I will move it there and update the types used in api/content-search as well

Reply:
I like that idea @jeremybarbet

Reply:
there is a  `libs/shared` with the two libs `form-fields` and `mocking`. Any preference on adding the `types` library there or in a new `libs/island-shared/` ?

Reply:
let's use `libs/shared` for this